web_src/js/features/repo-issue-content.js

```diff
@@ -79,7 +76,7 @@ function showContentHistoryMenu(issueBaseUrl, $item, commentId) {
   const $headerLeft = $item.find('.comment-header-left');
   const menuHtml = `
   <div class="ui pointing dropdown top left content-history-menu" data-comment-id="${commentId}">
-    <a>&bull; ${i18nTextEdited} ${svg('octicon-triangle-down', 17)}</a>
+    &bull; <a>${i18nTextEdited} ${svg('octicon-triangle-down', 14, 'dropdown icon')}</a>
```

Comment:
Hmm, should the bullet really not be part of the link anymore?

Reply:
IIRC, in the first PR, there is no  underline.

Some later PRs make the underline appears, so here the `<a>` needs a `muted`, then no need to move the bullet out.

Reply:
`muted` only changes color, not underline. That said, I think only the "edited" should be highlighted, but currently the whitspace after also is.

Reply:
Updated:

<img width="109" alt="image" src="https://user-images.githubusercontent.com/115237/205241070-3a40eca8-72f4-4bea-8b68-eb7da0c67013.png">